src/tests/utils/Processing.test.js

```diff
@@ -0,0 +1,88 @@
+import temp from 'temp';
+import { loadExternalDependenciesPlayground, readConfigPlayground, fillTank } from '../../utils/Processing';
+import { RawFSH } from 'fsh-sushi/dist/import/RawFSH';
+import { FHIRDefinitions } from 'fsh-sushi/dist/fhirdefs';
+import * as loadModule from '../../utils/Load';
+import 'fake-indexeddb/auto';
+
+describe('Processing', () => {
+  temp.track();
+
+  describe('#readConfigPlayground', () => {
+    it('should hard code the preferred config', () => {
+      const config = readConfigPlayground();
+      expect(config.fhirVersion).toEqual(['4.0.1']);
+      expect(config.id).toEqual('fhir.us.minimal');
+      expect(config.FSHOnly).toBe(false);
+    });
+  });
+
+  describe('#loadExternalDependenciesPlayground()', () => {
+    beforeAll((done) => {
```

Comment:
I'm not sure I've used this before. Can you explain what the `beforeAll` and `afterAll` are doing here so I know once you leave?

Reply:
This, like the manual long timeout on the end of the tests were leftover attempts at getting the tests to not timeout. (It didn't end up working) I removed it!

Reply:
Ah that makes sense! Thanks!